packages/client/src/generation/TSClient/helpers.ts

```diff
@@ -81,8 +78,7 @@ export function getArgFieldJSDoc(
   const fieldName = typeof field === 'string' ? field : field.name
   if (JSDocs[action] && JSDocs[action]?.fields[fieldName]) {
     const singular = type.name
```

Comment:
```suggestion
    const singular = type.name
```

I don't think this is used anymore